src/Models/Repository/FeedRepository.cs

```diff
@@ -0,0 +1,103 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Phytime.Models
+{
+    public class FeedRepository : IRepository<Feed, User>
```

Comment:
Why does your Feed repository know something about users and vise versa?
Your break SOLID, again

Or it's no repository logic, or it isn't FeedRepository